govcd/api_vcd_test.go

```diff
@@ -166,6 +166,11 @@ type TestConfig struct {
 			Size          int64 `yaml:"size,omitempty"`
 			SizeForUpdate int64 `yaml:"sizeForUpdate,omitempty"`
 		}
+		Nsxt struct {
+			Manager        string `yaml:"manager"`
+			Tier0router    string `yaml:"tier0router"`
+			Tier0routerVrf string `yaml:"tier0routerVrf"`
```

Comment:
in every config example `tier0routervrf` is used, but by code expected `tier0routerVrf`

Reply:
Fixed.